src/SqlParser.ts

```diff
@@ -423,7 +425,7 @@ class SqlParser extends CstParser {
   });
 
   public booleanExpressionValue = this.RULE("booleanExpressionValue", () => {
-    this.CONSUME(Identifier);
+    this.SUBRULE(this.columnPrimary);
```

Comment:
I fixed the grammar here, because this is valid SQL:

```sql
SELECT foodmart.account.account_id FROM "foodmart"."account" WHERE foodmart.account.account_id = foodmart.account.account_id
```